docs/src/pages/components/text-fields/ValidationTextFields.tsx

```diff
@@ -1,23 +1,17 @@
 import * as React from 'react';
+import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
-import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
-
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    root: {
-      '& .MuiTextField-root': {
-        margin: theme.spacing(1),
-        width: 200,
-      },
-    },
-  }),
-);
 
 export default function ValidationTextFields() {
-  const classes = useStyles();
-
   return (
-    <form className={classes.root} noValidate autoComplete="off">
+    <Box
+      component="form"
+      sx={{
+        '& .MuiTextField-root': { m: 1, width: '25ch' },
```

Comment:
```suggestion
        '& .MuiTextField-root': { m: 1, width: 200 },
```
?

Reply:
I believe `ch` was added by @eps1lon to fit with the base front size. A blame with the original PR should yield more details.

Reply:
Ah got it. Alright, let's resolve the comments that are not relevant. I think I added suggestions on all places where the values are different, at least to make sure we are not unintentionally changing the styles.

Reply:
If that change to `25ch` was a product of the visual in mobile so that all the TextFields would be seen of the same size. I will review each of the comments later.

Reply:
> If that change to 25ch was the product of a visual problem on mobile

It was added to guarantee that the textbox isn't squashed when increasing the browsers default font size. [The UI should not break until the default font size is increased to 200%](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F80).

Reply:
@eps1lon Thanks for the clarification and correction. I was only referring to the `width` property in my previous comment, I did not know the `ch`.